parser/scraper_halooglasi.py

```diff
@@ -54,13 +51,27 @@ def get_apartment_district(self, apartment) -> Optional[str]:
         places = [item.get_text(strip=True) for item in places_items if item.get_text(strip=True)]
         if places:
             return places[2]
```

Comment:
General rule we have - no magic numbers. In other cases you should create a constant for this number above or at least add a comment. Why 2? Just explain it somewhere 😄 (i'd prefer comment since this index is very local to this function).

Reply:
Added comments in this and other methods where "magic numbers" were present.